example1.tex

```diff
@@ -48,9 +48,8 @@
     \draw[->] (d) -- (V);
 \end{tikzpicture}
 
-Step 2: apply rules to eliminate $\distinct$ operators (the rules can
-be found in the full version of the paper), producing a sequence of
-equivalent circuits:
+Step 2: apply rules to eliminate $\distinct$ operators, producing a
+sequence of equivalent circuits:
```

Comment:
I think it's ok to skip intermediate versions and just show the final result of removing all distinct's since we don't explain those rules anyway